shared/ui/core/src/main/java/com/ivy/ui/FormatMoneyUseCase.kt

```diff
@@ -18,12 +23,27 @@ class FormatMoneyUseCase @Inject constructor(
     private val withoutDecimalFormatter = DecimalFormat("###,###", DecimalFormatSymbols(locale))
     private val withDecimalFormatter = DecimalFormat("###,###.00", DecimalFormatSymbols(locale))
 
-    suspend fun format(value: Double): String {
-        val showDecimalPoint = features.showDecimalNumber.isEnabled(context)
+    suspend fun format(value: Double, shortenAmount: Boolean): String {
```

Comment:
nit: Add KDoc on the function so other contributors will know how to use it

Reply:
Simplified the function itself, therefore haven't added KDoc, let me know if you'd still prefer KDoc

Reply:
It would be nice to have KDoc so users can see what the thing do from Android Studio and w/o opening the code. It's a blocker though, we can skip it if you're not comfortable with writing one. Simplifying is always nice!

Reply:
i.I'd prefer to do `KDoc ` for this in a separate issue.
ii.Also do we need additional logic for crypto? If so, I'd add it in another issue
Please let me know your comment on above

Reply:
Sure, sounds good to me. Crypto + KDoc in a separate issue